test/Microsoft.AspNetCore.Razor.Evolution.Test/IntegrationTests/IntializeTestFileAttribute.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.IntegrationTests
+{
+    public class IntializeTestFileAttribute : BeforeAfterTestAttribute
+    {
+        public override void Before(MethodInfo methodUnderTest)
+        {
+            if (typeof(IntegrationTestBase).IsAssignableFrom(methodUnderTest.DeclaringType))
+            {
+                var typeName = methodUnderTest.DeclaringType.Name;
+                IntegrationTestBase.Filename = $"TestFiles/IntegrationTests/{typeName}/{methodUnderTest.Name}";
```

Comment:
Hmm. We might want to separate the IR baselines from the incoming code gen baselines.